EPS/eps_subsystem.py

```diff
@@ -26,211 +26,116 @@
 """
 
 import sys
-import socket
+sys.path.append("../")
+from socket_stuff import create_socket_and_listen   # pylint: disable=C0413
+from command_handler import CommandHandler          # pylint: disable=C0413
+from command_factory import CommandFactory          # pylint: disable=C0413
 
 
 DEFAULT_HOST = '127.0.0.1'
 DEFAULT_PORT = 1801
 COMMAND_DELIMITER = ':'
 
 default_eps_state = {
-    'Temperature': 32,         # in degrees C
-    'Voltage': 5.24,       # in volts
-    'Current': 1.32,       # in amps
-    'State': 'Charging',
-    'WatchdogResetTime': 24.0,       # in hours
+    'Temperature': 32,           # in degrees C
```

Comment:
Is this diff implying a TAB was added? I thought pylon always went the other way? (i.e. removing TABs)

Reply:
I just shifted the comments describing the units over to look better